README.md

```diff
@@ -1,70 +1,113 @@
 # Cloud Data Access Client
 
-This project is a reference implementation for how to consume files from S3 that have been produced by Guidewire's Cloud Data Access (CDA).  
+The Cloud Data Access Client (CDA Client) is a utility to download table data from an Amazon S3 bucket that has been populated by Guidewire's Cloud Data Access (CDA). The utility reads the .parquet files generated by CDA for each table, and can do one of the following:
 
-The example code can be used to jump start development of a custom implementation that can store CDA data to other storage formats (e.g., RDBMS Tables, Hadoop/Hive, JSON Files).
+- Convert them into human readable .csv files
+- Transfer them as .parquet files
```

Comment:
The parquet files that CDA client puts out are not identical to the files that CDA writes, so I think this is a bit misleading.
```suggestion
- Aggregate them into fewer `.parquet` files
```